internal/server/apparmor/instance_lxc.go

```diff
@@ -27,7 +27,7 @@ profile "{{ .name }}" flags=(attach_disconnected,mediate_deleted) {
   signal peer=@{profile_name},
 
   # Allow normal process handling
-  ptrace (readby),
+  ptrace (readby),  # codespell:ignore readby
```

Comment:
Those hints in the apparmor generated profiles seem less than ideal as that ends up on the actual end user system and gets fed to the apparmor parser.

Is there a way to put the comment above the entire template block rather than in-line within the profile?

Reply:
Doesn't seem to be possible. I'm going to just split the actual profiles into separate files that we can then just ignore.

Reply:
Done